py/desispec/scripts/proc_joint_fit.py

```diff
@@ -308,7 +308,12 @@ def main(args=None, comm=None):
                 for flat in inflats_for_camera[camera]:
                     cmd += f" {flat} "
                 num_cmd += 1
-                err = runcmd(cmd, inputs=inflats_for_camera[camera], outputs=[ofile, ])
+                cmdargs = cmd.split()[1:]
+                average_fiberflat_args = desispec.scripts.average_fiberflat.parse(cmdargs)
+                try:
+                    err = runcmd(desispec.scripts.average_fiberflat.main, args=[average_fiberflat_args], inputs=inflats_for_camera[camera], outputs=[ofile, ])
```

Comment:
Minor: Stylistically I like this form where you explicitly call `runcmd` with a function name:
```
runcmd(desispec.scripts.average_fiberflat.main, ...)
```
instead of elsewhere which has a pattern of redefining `cmd` e.g.
```
cmd = "desi_blat_foo ..."
...
cmd = desispec.scripts.procexp.main
runcmd(cmd, ...)
```
The first approach makes it clearer when reading the `runcmd` line that it is being called with a function instead of a string, and avoids redefining what `cmd` means within the local block of code.

Reply:
I've changed the relevant `runcmd` calls in this way.